Mobile/ShopLens/Droid/Activities/CameraActivity.cs

```diff
@@ -100,6 +102,10 @@ protected override void OnCreate(Bundle savedInstanceState)
 
                 RecVoice = FindViewById<Button>(Resource.Id.btnRecVoiceCamera);
                 RecVoice.Click += RecogniseVoice;
+
+                shoppingCartErrorDialog = new ErrorDialogCreator(this, "Shopping cart", "Would you like to add this product to your shopping cart?",
+                    "Yes", "No", addToShoppingCart, doNotAddToShoppingCart);
+                shoppingCartMessageBar = new MessageBarCreator(rootView, "Product was added.");
```

Comment:
I would move the strings from 106, 107, 108 lines to app.config or Strings.xml file.